core/src/main/java/amino/run/kernel/metric/NodeMetric.java

```diff
@@ -0,0 +1,21 @@
+package amino.run.kernel.metric;
+
+import amino.run.kernel.client.KernelClient;
+import amino.run.kernel.common.ServerInfo;
+import java.io.Serializable;
+
+/**
+ * This class is used to hold the local kernel server metric relative to a remote kernel server.
+ * Local kernel server collects metrics w.r.t all remote kernel servers in {@link
+ * KernelClient#measureMetrics(KernelClient.KernelServerInfo)} and sends to OMS in the heartBeats at
+ * {@link amino.run.kernel.server.KernelServerImpl#sendHeartBeat(ServerInfo)}
+ */
+public class NodeMetric implements Serializable {
+    public long latency; // Latency between kernel servers in nanoseconds
+    public double rate; // Data transfer rate between kernel servers in Mbps
```

Comment:
I think rather keep this simple, make this a long and use bytes per second.  Stay away from double precision arithmetic etc.

Reply:
Changed rate field to long. And it is represented in bytes/sec now.